websites/D/Dek-d/metadata.json

```
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 2,
```

This looks like an api version 1 activity
```suggestion
	"apiVersion": 1,
```